ingestion/src/metadata/ingestion/source/dashboard/mstr/metadata.py

```diff
@@ -106,40 +106,41 @@ def yield_dashboard(
         """
         Method to Get Dashboard Entity
         """
-        try:
-            dashboard_url = (
-                f"{clean_uri(self.service_connection.hostPort)}/MicroStrategyLibrary/app/"
-                f"{dashboard_details.projectId}/{dashboard_details.id}"
-            )
-            dashboard_request = CreateDashboardRequest(
-                name=EntityName(dashboard_details.id),
-                displayName=dashboard_details.name,
-                sourceUrl=SourceUrl(dashboard_url),
-                project=dashboard_details.projectName,
-                charts=[
-                    FullyQualifiedEntityName(
-                        fqn.build(
-                            self.metadata,
-                            entity_type=Chart,
-                            service_name=self.context.get().dashboard_service,
-                            chart_name=chart,
+        if dashboard_details:
```

Comment:
What are the cases for this variable to not exist? Is it expected? Maybe better to log it so that we debug ingestions that "succeed but do nothing".